Apps/phadldap/adldap.json

```diff
@@ -0,0 +1,1311 @@
+                    "data_type": "string",
+                    "column_name": "current status",
+                    "column_order": 2,
+                    "example_values": [
+                        "enabled"
+                    ]
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.use_samaccountname",
+                    "data_type": "boolean",
+                    "example_values": [
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "Account status: enabled"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.user_dn",
+                    "data_type": "string",
+                    "example_values": [
+                        "cn=defaultaccount,cn=users,dc=splunk,dc=lab"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            }
+        },
+        {
+            "action": "reset password",
+            "identifier": "reset_password",
+            "description": "Reset a user's password",
+            "type": "custom",
+            "read_only": false,
+            "versions": "EQ(*)",
+            "parameters": {
+                "use_samaccountname": {
+                    "description": "Specify samaccountname instead of distinguishedname",
+                    "data_type": "boolean",
+                    "required": false,
+                    "primary": false,
+                    "value_list": [],
+                    "default": false,
+                    "order": 0
+                },
+                "user": {
+                    "description": "Specify the dn to enable",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "user name"
+                    ],
+                    "default": false,
+                    "order": 1
+                },
+                "password": {
```

Comment:
It might also be good to have a retype password text area if you can mask the password. If password and retyped-password match, then set the new password. Else, fail action

Reply:
Good idea, adding.